web-app/js/portal/filter/BaseFilterPanel.js

```diff
@@ -84,6 +84,14 @@ Portal.filter.BaseFilterPanel = Ext.extend(Ext.Panel, {
     },
 
     _setExistingFilters: function() {
+    },
+
+    needsFilterRange: function() {
```

Comment:
Oh, maybe this is unclear. The filter UI is notified using the setFilterRange(range) after he range is loaded. The needsFilterRange() method just lets you know whether the UI is going to use the range or not.